solutions/instances/outputs.tf

```diff
@@ -14,22 +14,22 @@ output "resource_group_id" {
 
 output "scc_id" {
   description = "SCC instance ID"
-  value       = module.scc.id
+  value       = var.existing_scc_instance_crn == null ? module.scc[0].id : null
 }
 
 output "scc_guid" {
   description = "SCC instance guid"
-  value       = module.scc.guid
+  value       = var.existing_scc_instance_crn == null ? module.scc[0].guid : null
```

Comment:
Existing instance GUID can be parsed from the `existing_scc_instance_crn` value

Reply:
addressed